### PR TITLE
[DebugInfo] Update tests to use opaque ptrs

### PR DESCRIPTION
**Pull request by felipepiovezan**
Update all /DebugInfo tests to use opaque pointers

### CODE REVIEW

**Comment by felipepiovezan** [comment]
@swift-ci test

**Comment by felipepiovezan** [comment]
@swift-ci test

**Review by aschwaighofer — APPROVED** [review]
Thank you

File: test/DebugInfo/struct_resilience.swift [hunk 1]
```diff
@@ -6,15 +6,13 @@
 // RUN:   -emit-module-path=%t/resilient_struct.swiftmodule \
 // RUN:   -module-name=resilient_struct %S/../Inputs/resilient_struct.swift
 //
-// RUN: %target-swift-frontend %use_no_opaque_pointers -g -I %t -emit-ir -enable-library-evolution %s \
-// RUN:    -o - | %FileCheck %s
 // RUN: %target-swift-frontend -g -I %t -emit-ir -enable-library-evolution %s \
-// RUN:    -o -
+// RUN:    -o - | %FileCheck %s
```
Comment (adrian-prantl) [inline test/DebugInfo/struct_resilience.swift:10]: Nice catch!

**Review by adrian-prantl — APPROVED** [review]
Thanks!

**Pull request merged**